projectDocs/issues/triage.md

```diff
@@ -0,0 +1,178 @@
+- Knowing who, helps to give an estimate on how many users this will help.
+- It also can help to highlight differences in requirements for different users. This happens when we are unable to define the same use case for two groups of users.
+
+### What
+
+- This can be a step by step of what they expect to do, and the kind of output they expect along the way.
+- Things to consider here:
+ - Does this need to work with other software? If so, what version?
+ - Particularly when working with other software, it's helpful if an example document or file can be provided. Perhaps a relevant test case?
+
+### Why
+Often the hardest one, but also the most valuable.
+
+- Why do they want to do this?
+- What does it help them to achieve?
+- It's valuable because it brings further understanding about the background that led to the what.
+Perhaps once we have this background, a simpler what can be proposed.
+
+## Summarise the issue
+It's common that this process of collecting information will result in many comments on the issue.
+As the GitHub issue grows, with more comments, questions, and discussion, it's useful to summarise the issues periodically.
+This helps to condense various back and forth discussions into the final result.
+This will make it easier for someone to quickly pick up the issue and understand it by reading the summary comment.
+This also serves to re-iterate decisions that have been made throughout the discussion and ensure that everyone is on the same page.
+
+## Labelling
+NV Access can grant people who help triage issues the ability to label issues.
+Labelling issues help indicate the priority and current state, helping NV Access and the community to decide on how to prioritise it.
+
+### Types of issues
+Issues can generally be labelled `bug` or `feature`.
+We also have a label for `enhancement`, think of this as a more internal facing change. For instance, editing code comments to provide clearer / more complete information, or extending an internal framework/API to unblock other issues.
+
+### Triaged status
+An issue is triaged if it is ready to be worked on.
+Once a bug has clear steps to reproduce and is well documented, the `triaged` label can be applied.
+New features and enhancements should be [well defined](#new-features-1) before applying the `triaged` label.
+
+If it is a complex issue, technical investigation may be required. This can be indicated with adding the label `blocked/needs-technical-investigation`.
+
+A `triaged` issue that requires a complex fix may require advice from NV Access, such as a project plan, before implementation is started.
+An issue with a simple solution should get labelled `good first issue`.
+
+For controversial changes, a product decision from NV Access may be required before applying the `triaged` label. This can be indicated with adding the label `blocked/needs-product-decision`.
+
+### Priority

```

Comment:
I think it would really help to have a per priority break down of what happens when an issue with that priority pops up in several stages of development. Some random examples:

- P1: Indicates that the issue is serious enough that it has to be assigned to the milestone of the current beta release cycle (if any) or alpha otherwise. If a release has just been made, a hotfix should be considered
- P2, assigned to the milestone of current master cycle, pushed forward to next milestone when it is not feasible for the current
Etc. Etc.

Reply:
I think we will have to monitor the frequency of P2s to be able to determine what exactly we can commit to.
With limited capacity we can really only consider this a priority system.
With these new definitions we wanted to make maintaining 0 P1s more possible, by limiting the definition of a P1 further. We currently have many P1s that are rare/hard to reproduce issues.